x-pack/test/fleet_api_integration/apis/package_policy/upgrade.ts

```diff
@@ -1360,7 +1360,10 @@ export default function (providerContext: FtrProviderContext) {
           expectIdArraysEqual(installation.installed_es, expectedAssets);
 
           for (const expectedAsset of expectedAssets) {
-            if (expectedAsset.type === 'component_template') {
+            if (
```

Comment:
wouldn't it be simpler to remove `@custom` from `expectedAssets` array?

Reply:
Yes I will update that test 👍